UKFast.API.Client.ECloud/Models/ActiveDirectoryDomain.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UKFast.API.Client.Models;
+
+namespace UKFast.API.Client.ECloud.Models
+{
+	public class ActiveDirectoryDomain : ModelBase
+	{
```

Comment:
Can you convert tabs to spaces here please :)

Reply:
Changed :)